compiler-rt/test/builtins/Unit/floatunditf_test.c

```diff
@@ -12,7 +12,7 @@
 
```

Comment:
Can we get rid of LDBL_MANT_DIG == 113 guard here? The test won't run for x86 platforms where CRT_HAS_TF_MODE is true but LDBL_MANT_DIG is not 113.

Reply:
Yes, this should just be a defined(CRT_LDBL_IEEE_F128)